components/HeaderBlock.vue

```diff
@@ -1,13 +1,20 @@
 <template>
-  <header class="container flex-row-reverse py-4 md:flex md:flex-row">
-    <div>
-      <client-only>
-        <RadioPlayer :show-album-art="true" :now-playing-uri="streamServer" />
-      </client-only>
+  <header class="flex-row-reverse items-center justify-between px-4 py-4 bg-white md:flex md:flex-row">
```

Comment:
Wow! I learned a lot looking these classes up.

Reply:
it's tailwind css classes for styling, similar to bootstrap (but much nicer and modern)